torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -1991,17 +1993,50 @@ def _backward_impl(ctx, all_args):
                     context = torch._C._DisableAutocast if disable_amp else nullcontext
                     with tracing(saved_context), compile_context(
                         saved_compile_context
-                    ), context(), track_graph_compiling(aot_config, "backward"):
-                        CompiledFunction.compiled_bw = aot_config.bw_compiler(
-                            bw_module, placeholder_list
-                        )
-                        # Maybe save cache entry
-                        if try_save_cache_entry is not None:
-                            try_save_cache_entry(
-                                CompiledFunction.compiled_bw,
-                                fw_metadata,
-                                aot_config,
+                    ), context(), track_graph_compiling(
+                        aot_config, "backward"
+                    ), METRICS_CONTEXT, dynamo_timed(
+                        # TODO: Capture this under "entire_frame_compile" or a new name?
```

Comment:
Please comment on this TODO. Note that this affects how we accumulate the timings that we report from `print_time_report()`. Previously we'd see something like:

```
TIMING: code_gen:0.82134 inductor_compile:0.92101 backend_compile:3.53894 entire_frame_compile:3.61238 total_wall_time:4.43129
```
With this change, we'd see something like:
```
TIMING: code_gen:0.67811 inductor_compile:0.774 backend_compile:2.73924 entire_frame_compile:3.48979 total_wall_time:3.48979
```
That is, the entire_frame_compile now includes the backward compile. I think that's better, but I wonder if it's actually more confusing.

Reply:
No, you can't put it in `entire_frame_compile`. If you start populating it from this site, we now have no way of distinguishing how much time was spent in Dynamo. To do a full summation, we should use the `duration_us` column, which is ALWAYS populated (well, it's not currently populated in dynamo_timed, but it should be.)